node/peer_score_strategies.js

```diff
@@ -64,9 +64,6 @@ PreferOutgoing.prototype.getScore = function getScore() {
     self.lastTier = tier;
     switch (tier) {
         case PreferOutgoing.ONLY_INCOMING:
-            if (!self.peer.channel.destroyed) {
-                self.peer.connectTo();
```

Comment:
this looks like a separate change from the "use it in hyperbahn intent", please split it out.